source/gui/settingsDialogs.py

```diff
@@ -1791,6 +1791,61 @@ def makeSettings(self, settingsSizer):
 				break
 		else:
 			log.debugWarning("Could not set progress list to current report progress bar updates setting")
+		self.progressList.Bind(wx.EVT_CHOICE, self.onProgressBarOutputModeChange)
+
+		# Translators: The label for a setting in presentation settings to change speech percentage interval
+		SpeechIntrLabelText = _("Progress bar &speech percentage Interval")
+		minSpeechIntr = int(config.conf.getConfigValidation(
+			("presentation", "progressBarUpdates", "speechPercentageInterval")
+		).kwargs["min"])
+		maxSpeechIntr = int(config.conf.getConfigValidation(
+			("presentation", "progressBarUpdates", "speechPercentageInterval")
+		).kwargs["max"])
+		self.SpeechIntrEdit = sHelper.addLabeledControl(
+			SpeechIntrLabelText,
+			nvdaControls.SelectOnFocusSpinCtrl,
+			min=minSpeechIntr,
+			max=maxSpeechIntr,
+			initial=config.conf["presentation"]["progressBarUpdates"]["speechPercentageInterval"]
+		)
+		if self.progressList.GetSelection() == 0 or self.progressList.GetSelection() == 2:
+			self.SpeechIntrEdit.Disable()
+
+		# Translators: The label for a setting in presentation settings to change beep percentage interval
+		BeepIntrLabelText = _("Progress bar b&eep percentage Interval")
+		minBeepIntr = int(config.conf.getConfigValidation(
+			("presentation", "progressBarUpdates", "beepPercentageInterval")
+		).kwargs["min"])
+		maxBeepIntr = int(config.conf.getConfigValidation(
+			("presentation", "progressBarUpdates", "beepPercentageInterval")
+		).kwargs["max"])
+		self.BeepIntrEdit = sHelper.addLabeledControl(
+			BeepIntrLabelText,
+			nvdaControls.SelectOnFocusSpinCtrl,
+			min=minBeepIntr,
+			max=maxBeepIntr,
+			initial=config.conf["presentation"]["progressBarUpdates"]["beepPercentageInterval"]
+		)
+		if self.progressList.GetSelection() == 0 or self.progressList.GetSelection() == 1:
+			self.BeepIntrEdit.Disable()
+
+		# Translators: The label for a setting in presentation settings to change minimum beep frequency
+		beepMinHZLabelText = _("Progress bar minimum beep &frequency (Hz)")

```

Comment:
I"m not sure whether I'd agree with adding this to the gui. Expressing pitch in terms of frequency sounds rather technical. I think I'd move this to the advanced settings panel or remove it again. Thoughts @feerrenrut?